### ISSUE TITLE
doctrine/annotations version conflict

### ISSUE THREAD

**Issue by MartkCz** [nettrine/annotations#5]
I want try nettrine instead of kdyby, but I have composer error.

My composer.json
```json
"nettrine/orm": "dev-master",
"nettrine/dbal": "dev-master",
"nettrine/annotations": "dev-master",
```

Error:
```
    - Installation request for nettrine/orm dev-master -> satisfiable by nettrine/orm[dev-master].
    - Can only install one of: doctrine/annotations[v1.6.0, v1.4.0].
    - nettrine/annotations dev-master requires doctrine/annotations 1.4.0 -> satisfiable by doctrine/annotations[v1.4.0].
    - nettrine/orm dev-master requires doctrine/annotations ~1.6.0 -> satisfiable by doctrine/annotations[v1.6.0].
    - Installation request for nettrine/annotations dev-master -> satisfiable by nettrine/annotations[dev-master].

```

**Comment by f3l1x** [comment]
Hi, do you really need nettrine/annotations? This package is suitable for writing your own annotations. 

There is a special extension just for ORM.

Take a look - https://github.com/nettrine/orm/tree/master/.docs#minimal-configuration

```yaml
extensions:
    orm.annotations: Nettrine\ORM\DI\OrmAnnotationsExtension

orm.annotations:
    paths:
        - App/Model/Database/Entity
```

**Comment by enumag** [comment]
@f3l1x Either way nettrine/annotations should not be locked to one specific version of doctrine/annotations. [This](https://github.com/nettrine/annotations/blob/master/composer.json#L16) should probably be `^1.4.0` instead.

**Comment by f3l1x** [comment]
You're right guys. I will move it to PHP 7.1 and unlock the version.

**Comment by JanMikes** [comment]
Hi @f3l1x any progress on this please? We are migrating from kdyby packages to nettrine+contributte, use own annotations and have same problem:
```
Problem 1
    - Installation request for nettrine/orm ^0.1 -> satisfiable by nettrine/orm[v0.1].
    - nettrine/orm v0.1 requires doctrine/annotations ^1.6.0 -> satisfiable by doctrine/annotations[v1.6.0].
    - Can only install one of: doctrine/annotations[v1.6.0, v1.4.0].
    - Can only install one of: doctrine/annotations[v1.4.0, v1.6.0].
    - nettrine/annotations v0.1 requires doctrine/annotations 1.4.0 -> satisfiable by doctrine/annotations[v1.4.0].
    - Installation request for nettrine/annotations ^0.1 -> satisfiable by nettrine/annotations[v0.1].

```

**Comment by f3l1x** [comment]
Should be fixed in: 8e9a1b8ef4c8149f47c562aea1546585d10db426
